### DISCUSSION TITLE
Multiple Submit Buttons with Different Actions

### DISCUSSION THREAD

**Discussion by vasuhooda** [solspace/craft-freeform#315] [Questions & Answers] [answered]
The plugin allows the user to add multiple submit buttons.  What is the use of this? 

We want to add two submit buttons, each button should redirect to a different URL. Is it possible to do it with the plugin?

**Comment by kjmartens** [comment, marked as answer]
Hi @vasuhooda, this was just set up to allow the option to potentially include a submit button in multiple locations (e.g. one at the top of the form, one at the bottom). Obviously not a common use case, but we didn't see any harm in allowing for it to happen. 🙂 

As for having different submit buttons that direct the user to a different URL, this is not currently possible. However, there is a way to handle this, perhaps in an even better way. Check out the **User Chooses Return URL** section of the [Success Return URL on Forms](https://docs.solspace.com/craft/freeform/v3/guides/success-return-on-forms.html#user-chooses-return-url) guide. You can manipulate the form at template level to have a field control the return URL (could be a checkbox or select dropdown, etc). 🙂

**Comment by vasuhooda** [comment]
@kjmartens 

Can I use AJAX and `freeform-render-success` to achieve the desired results?

**Reply by kjmartens** [reply]
You would have to create a module that handles different submit buttons and modify the return URL that way.
